NOTES-Valgrind.md

```diff
@@ -14,34 +14,33 @@ Requirements
 ------------
 
 1. Platform supported by Valgrind
-   See: http://valgrind.org/info/platforms.html
+   See http://valgrind.org/info/platforms.html
 2. Valgrind installed on the platform
-   See: http://valgrind.org/downloads/current.html
-3. OpensSSL compiled
-   See: INSTALL
+   See http://valgrind.org/downloads/current.html
+3. OpensSSL compiled, sSee [INSTALL.md](INSTALL.md)
 
 Running Tests
 -------------
 
 Test behavior can be modified by adjusting environment variables.
 
-EXE_SHELL
+    EXE_SHELL
```

Comment:
This indentation is ugly, please don't.

Reply:
> This indentation is ugly, please don't.

I did that to get code block formatting, but changed it to 
```
`EXE_SHELL`
```
also for similar cases below.

Reply:
I totally understand the intention, but we must keep in mind that people are reading these files with `more` or `less` as well, so there's a fine line to walk to get it as readable as possible, as straight text as well as with MD rendition.